geode-core/src/main/java/org/apache/geode/management/cli/ConverterHint.java

```diff
@@ -44,6 +44,8 @@
   String REGION_PATH = "geode.converter.region.path" + DISABLE_STRING_CONVERTER;
   String INDEX_TYPE = "geode.converter.index.type" + DISABLE_ENUM_CONVERTER;
   String GATEWAY_SENDER_ID = "geode.converter.gateway.senderid" + DISABLE_STRING_CONVERTER;
+  String ASYNC_EVENT_QUEUE_ID =
```

Comment:
If you don't have a converter implemented for this event queue id to provide some auto-completion, then you don't need this converter hint.

Reply:
That makes sense. We've removed the ConverterHint from this command.